src/type/__tests__/introspection-test.ts

```diff
@@ -1519,6 +1536,105 @@ describe('Introspection', () => {
     });
   });
 
+  it('identifies oneOf for input objects', () => {
+    const schema = buildSchema(`
+      input SomeInputObject @oneOf {
+        a: String
+      }
+
+      input AnotherInputObject {
+        a: String
+        b: String
+      }
+
+      type Query {
+        someField(someArg: SomeInputObject): String
+        anotherField(anotherArg: AnotherInputObject): String
+      }
+    `);
+
+    const source = `
+      {
+        a: __type(name: "SomeInputObject") {
+          isOneOf
+        }
+        b: __type(name: "AnotherInputObject") {
```

Comment:
nit: please use meaningful names for types and aliases as in other tests.
`oneOfObject: __type(name: "OneOfObject")`